pages/erd.vue

```diff
@@ -1,18 +1,44 @@
 <template>
-  <div id="erd">
-    <VideoPlayer src="video/erd_video.mp4"></VideoPlayer>
-  </div>
+  <v-container>
+    <v-row>
+      <v-col cols="12">
+        <v-btn color="primary" @click="goBack">Back</v-btn>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <div id="erd">
+          <VideoPlayer :src="'../videos/' + video"></VideoPlayer>
+        </div>
+      </v-col>
+    </v-row>
+  </v-container>
 </template>
 
 <script>
-import VideoPlayer from "../components/VideoPlayer";
+import VideoPlayer from '../components/VideoPlayer'
```

Comment:
it is a good practice to use relative path  '~/components/VideoPlayer'
which refers to the root of the project, and it can help later for refactoring.